package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "multi-select-react",
-  "version": "0.1.8",
+  "version": "0.2.1",
```

Comment:
Try not to change version in package.json, this is how we can keep track of versions. Branch name should be same as version in the package.json.